tests/framework/test_util.h

```diff
@@ -512,6 +512,8 @@ inline std::ostream& operator<<(std::ostream& os, const VkResult& result) {
             return os << "VK_ERROR_VIDEO_PROFILE_CODEC_NOT_SUPPORTED_KHR";
         case (VK_ERROR_VIDEO_STD_VERSION_NOT_SUPPORTED_KHR):
             return os << "VK_ERROR_VIDEO_STD_VERSION_NOT_SUPPORTED_KHR";
+        case (VK_ERROR_INVALID_VIDEO_STD_PARAMETERS_KHR):
+            return os << "VK_ERROR_INVALID_VIDEO_STD_PARAMETERS_KHR";
```

Comment:
Good catch!
Though, I feel like this should be generated. But since its for tests (and few of the error codes are used by the loader) its fine for it to be manually modified.